packages/ubuntu_bootstrap/test/storage/storage_page_test.dart

```diff
@@ -32,112 +34,138 @@ void main() {
-    final model = buildStorageModel(existingOS: [
-      const OsProber(long: 'Windows 10', label: 'windows', type: 'ntfs'),
-      const OsProber(long: 'Ubuntu 20.04 LTS', label: 'Ubuntu', type: 'ext4'),
-      const OsProber(long: 'Ubuntu 20.04 LTS', label: 'Ubuntu', type: 'ext4')
-    ]);
+    const productInfo = ProductInfo(name: 'Ubuntu');
+    final model = buildStorageModel(
+      existingOS: [
+        const OsProber(long: 'Windows 10', label: 'windows', type: 'ntfs'),
+        const OsProber(long: 'Ubuntu 20.04 LTS', label: 'Ubuntu', type: 'ext4'),
+        const OsProber(long: 'Ubuntu 20.04 LTS', label: 'Ubuntu', type: 'ext4')
+      ],
+      productInfo: productInfo,
+    );
     await tester.pumpApp((_) => buildPage(model));
 
     final context = tester.element(find.byType(StoragePage));
     final l10n = UbuntuBootstrapLocalizations.of(context);
 
     expect(
-      find.text(l10n.installationTypeMultiOSDetected),
+      find.text(l10n.installationTypeAlongsideMulti(productInfo.name)),
       findsOneWidget,
     );
   });
 
   testWidgets('duplicate existing OSes', (tester) async {
-    final model = buildStorageModel(existingOS: [
-      const OsProber(long: 'Ubuntu 20.04 LTS', label: 'Ubuntu', type: 'ext4'),
-      const OsProber(long: 'Ubuntu 20.04 LTS', label: 'Ubuntu', type: 'ext4')
-    ]);
+    const productInfo = ProductInfo(name: 'Ubuntu');
+    final model = buildStorageModel(
+      existingOS: [
+        const OsProber(long: 'Ubuntu 20.04 LTS', label: 'Ubuntu', type: 'ext4'),
+        const OsProber(long: 'Ubuntu 20.04 LTS', label: 'Ubuntu', type: 'ext4')
+      ],
+      productInfo: productInfo,
+    );
     await tester.pumpApp((_) => buildPage(model));
 
     final context = tester.element(find.byType(StoragePage));
     final l10n = UbuntuBootstrapLocalizations.of(context);
 
     expect(
-      find.text(l10n.installationTypeMultiOSDetected),
+      find.text(l10n.installationTypeAlongsideMulti(productInfo.name)),
       findsOneWidget,
     );
   });
 
   testWidgets('alongside windows', (tester) async {
+    const productInfo = ProductInfo(name: 'Ubuntu 22.10');
+    const osProber = OsProber(
+      long: 'Windows 10',
+      label: 'WIN10',
+      version: '10',
+      type: 'ntfs',
+    );
     final model = buildStorageModel(
-      productInfo: ProductInfo(name: 'Ubuntu 22.10'),
-      existingOS: [
-        const OsProber(
-          long: 'Windows 10',
-          label: 'WIN10',
-          version: '10',
-          type: 'ntfs',
-        ),
-      ],
+      productInfo: productInfo,
+      existingOS: [osProber],
       canInstallAlongside: true,
     );
     await tester.pumpApp((_) => buildPage(model));
 
     final context = tester.element(find.byType(StoragePage));
     final l10n = UbuntuBootstrapLocalizations.of(context);
 
-    final radio = find.radioButton<StorageType>(
-        l10n.installationTypeAlongside('Ubuntu 22.10', 'Windows 10'));
+    final radio = find.text(
```

Comment:
Does `find.radio()` work here?

Reply:
Unfortunately not since the title isn't in the `YaruRadio`, but directly in the `YaruRadioListTile`

Reply:
I can do this instead:
```dart
find.widgetWithText(
      YaruRadioListTile<StorageType>,
      l10n.installationTypeAlongside(productInfo.name, osProber.long),
    );
```
I'll make a local helper for that

Reply:
Done in https://github.com/canonical/ubuntu-desktop-provision/pull/478/commits/4e7b112fd189ae8dffe4768a0f0fcb55b1e9da73